tests/util.rs

```diff
@@ -0,0 +1,14 @@
+use std::fs::OpenOptions;
+use std::io;
+use std::io::prelude::*;
+use std::path::Path;
+
+/// Creates a file at `path` and writes binary `data` to it.
+///
+/// This can be used to create binary bincode data files as is done in
+/// `rust-bitcoin/tests/data/serde`. Pass in the output of serde serialization as `data`.
+pub fn create_bincode_file<P: AsRef<Path> + ?Sized>(data: &[u8], path: &P) -> Result<(), io::Error>{
+    let mut file = OpenOptions::new().write(true).create_new(true).open(path)?;
+    file.write_all(&data)?;
+    Ok(())
+}
```

Comment:
This writes arbitrary data, not specifically bincode and looks like the only difference from `std::fs::write` is `create_new(true)`. I wonder if it being present could create more noise than value...

Reply:
lol, I thought this exact thing myself after trimming the function down.  While writing a response just now I realized that it would be just as good to put the two lines of code into the comment at the head of the file and remove this all together. Sorry for the noise, totally should have thought of that before.

Reply:
No problem!